### ISSUE TITLE
Failed to fetch unauthenticated user on chrome

### ISSUE THREAD

**Issue by AstroneCommons** [aws-amplify/amplify-js#6572]
**Describe the bug**
`currentCredentials` returns a `failed to fetch` error
due to net::ERR_BLOCKED_BY_RESPONSE

**To Reproduce**

1. call `Auth.currentCredentials() ` without being authenticated on chrome 84
4. See `error: Failed to fetch` 

Debug logs

```
[DEBUG] 30:51.530 AuthClass - getting current credentials
ConsoleLogger.js:97 [DEBUG] 30:51.530 Credentials - getting credentials
ConsoleLogger.js:97 [DEBUG] 30:51.530 Credentials - picking up credentials
ConsoleLogger.js:97 [DEBUG] 30:51.530 Credentials - getting new cred promise
ConsoleLogger.js:97 [DEBUG] 30:51.530 Credentials - checking if credentials exists and not expired
ConsoleLogger.js:97 [DEBUG] 30:51.530 Credentials - need to get a new credential or refresh the existing one
ConsoleLogger.js:97 [DEBUG] 30:51.530 AuthClass - Getting current user credentials
ConsoleLogger.js:107 [DEBUG] 30:51.531 AuthClass - failed to get or parse item aws-amplify-federatedInfo SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at AuthClass.<anonymous> (Auth.js:1270)
    at step (Auth.js:44)
    at Object.next (Auth.js:25)
    at fulfilled (Auth.js:16)
ConsoleLogger.js:97 [DEBUG] 30:51.531 AuthClass - Getting current session
ConsoleLogger.js:97 [DEBUG] 30:51.537 Credentials - getting credentials
ConsoleLogger.js:97 [DEBUG] 30:51.537 Credentials - picking up credentials
ConsoleLogger.js:97 [DEBUG] 30:51.537 Credentials - getting old cred promise
ConsoleLogger.js:97 [DEBUG] 30:51.642 AuthClass - Failed to get user from user pool
ConsoleLogger.js:107 [DEBUG] 30:51.644 AuthClass - Failed to get the current user No current user
ConsoleLogger.js:107 [DEBUG] 30:51.644 AuthClass - getting session failed No current user
ConsoleLogger.js:97 [DEBUG] 30:51.644 Credentials - setting credentials for guest
VM12:1 POST https://cognito-identity.eu-west-1.amazonaws.com/ net::ERR_BLOCKED_BY_RESPONSE
ConsoleLogger.js:107 [DEBUG] 30:52.85 Credentials - Failed to load credentials Promise {<rejected>: TypeError: Failed to fetch}
ConsoleLogger.js:107 [DEBUG] 30:52.85 Credentials - Error loading credentials TypeError: Failed to fetch
App.tsx:87 Credentials:  TypeError: Failed to fetch
ConsoleLogger.js:107 [DEBUG] 30:52.85 AWSS3Provider - set credentials for storage {accessKeyId: undefined, sessionToken: undefined, secretAccessKey: undefined, identityId: undefined, authenticated: undefined}
ConsoleLogger.js:97 [DEBUG] 30:52.86 AWSS3Provider - get test.txt from public/test.txt
```

**Expected behavior**
Auth.currentCredentials should work with unauthenticated user

The error seems inconsistent it temporally works:
by clearing the browser cache 
and in incognito mode

**Comment by AstroneCommons** [comment]
Steps to reproduce:
1- use chrome after version 83
2- clear the cache
3- set the following response headers:
      -"Cross-Origin-Opener-Policy": "same-origin",
      -"Cross-Origin-Embedder-Policy": "require-corp"
4- Call Auth.currentCredentials()

the fetch will fail with net::ERR_BLOCKED_BY_RESPONSE
Everything works fine with firefox 79

This means that unauth authentication is not supported by chrome in "same-site isolation" mode required to use sharredarraybuffer for example.

it seems that CORS are set by identity pool services, and COEP should not cause problem but it does.
amplify bug ?
chrome bug ?

**Comment by AstroneCommons** [comment]
I check the amplify source code and imho the fetch call is done well. But i could have missing something.
This is most likely a chromium bug.
net::ERR_BLOCKED_BY_RESPONSE is a new dev-tool message for Cross Origin Blocked Response (CORB).
As far as i known, CORB should not be triggered by cors mode fetch.

**Comment by AstroneCommons** [comment]
**confirmed chromium bug**

**Comment by eryon** [comment]
@AstroneCommons do you have a link to the chromium bug?

**Comment by BenoitDel** [comment]
here you go: [bug #1116990]( https://bugs.chromium.org/p/chromium/issues/detail?id=1116990#c21).

**Comment by AstroneCommons** [comment]
fixed with Version 86.0.4240.8.
The bug has been fixed by chrome team. Amplify team is still wondering if the bug can be reproduced.
